### ISSUE TITLE
Redirect is not redirecting from node/123 to path alias

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#2871]
## Describe your issue or idea
In Drupal there was both the globalredirect module, and a setting for the redirect module, that made it so the system-path for a page would never be used, instead the pretty path alias would be used instead.

### Steps to reproduce (if reporting a bug)
Create a node, give it a URL alias
Access it at the system path

### Actual behavior (if reporting a bug)
Note, no redirect is performed
### Expected behavior (if reporting a bug)
Redirect to path alias

**Comment by jlfranklin** [comment]
Redirect & globalredirect are both in the backdrop-contrib project.  If you install both of them, do you get the expected behavior?

**Comment by jenlampton** [comment]
Redirect is already in core. 

Installing the contrib version of globalredirect does exactly as it's supposed to, and creates the behavior I had expected from core alone.

**Comment by olafgrabienski** [comment]
Doesn't it make sense to integrate the Global Redirect functionality into core's Redirect? There is a similar undertaking in Drupal's Redirect module, see https://www.drupal.org/node/905914.

**Comment by BWPanda** [comment]
From duplicate issue https://github.com/backdrop/backdrop-issues/issues/3264:

> When you have a URL alias setup for a node, you have two URLs by which to access the same content:
>
> - node/[n]
> - my/custom/alias
>
> Neither redirects to the other. This creates duplicate content which, according to [Moz](https://moz.com/learn/SEO/duplicate-content), may negatively affect your SEO.
>
> I recommend adding an option (since not everyone may want this turned on by default), that makes it so that accessing a raw node URL that has an alias, redirects to that alias instead. Alternatively, maybe as another option, we can add a rel=canonical tag to the raw node URL page, referencing the aliased URL.

Since this issue is about setting up automatic redirects, I'm happy with that and this has my vote.

**Comment by jenlampton** [comment]
Is this one another related/duplicate issue https://github.com/backdrop/backdrop-issues/issues/868?

**Comment by olafgrabienski** [comment]
Yes, #868 seems to be the same. Not sure about the common practice: close the newer issue (this one)?

**Comment by klonos** [comment]
Common practice is to close new in favor of old issues, but as long as there are no relevant comments in the newer issues. Since the comments in the old issue were made back in 2015/2016, I'd say close the old one.

**Comment by cptX** [comment]
This functionality (global redirect) is super critical and we should include it in the core. I think this issue should get much higher priority! It's 3.5 years old already!

**Comment by jenlampton** [comment]
Thanks for your comment @cptX! 

If you would like a feature request to get into Backdrop core, you might consider becoming an advocate for it: https://backdropcms.org/news/become-an-advocate-for-a-new-feature-in-backdrop-core

**Comment by BWPanda** [comment]
GlobalRedirect doesn't redirect comment links, so I created a feature request for that: https://github.com/backdrop-contrib/globalredirect/issues/11 But in case the module gets put into core before that's added, I'll leave https://github.com/backdrop/backdrop-issues/issues/3265 open.
